terraform/modules/monitoring/sg_monitoring.tf

```diff
@@ -8,44 +8,34 @@ resource "aws_security_group" "monitoring" {
   description = "Allow access to incoming monitoring traffic"
   vpc_id = "${var.vpc_id}"
 
-  tags =  {
+  tags {
     Name = "${var.stack_description} - Incoming Monitoring Traffic"
   }
-
 }
 
 resource "aws_security_group_rule" "self_reference" {
-    type = "ingress"
-    self = true
-    from_port = 0
-    to_port = 0
-    protocol = -1
-    security_group_id = "${aws_security_group.monitoring.id}"
-}
-
-resource "aws_security_group_rule" "monitoring_grafana" {
-    type = "ingress"
-    from_port = 3000
-    to_port = 3000
-    protocol = "tcp"
-    cidr_blocks = ["0.0.0.0/0"]
-    security_group_id = "${aws_security_group.monitoring.id}"
+  type = "ingress"
+  self = true
+  from_port = 0
+  to_port = 0
+  protocol = -1
+  security_group_id = "${aws_security_group.monitoring.id}"
 }
 
-resource "aws_security_group_rule" "monitoring_riemann" {
-    type = "ingress"
-    from_port = 5555
-    to_port = 5559
-    protocol = "all"
-    cidr_blocks = ["0.0.0.0/0"]
-    security_group_id = "${aws_security_group.monitoring.id}"
+resource "aws_security_group_rule" "monitoring_nginx" {
```

Comment:
Looks like this rule is being added, but not used. What is it for?

Reply:
This allows access from the load balancer to nginx on 8080. It's redundant with https://github.com/18F/cg-provision/blob/master/terraform/modules/bosh_vpc/sg_bosh.tf#L72-L79, but eventually I think we should drop those overgeneral rules from sg-bosh and move them into security groups that are specific to a deployment.